src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -207,7 +209,20 @@ public static void RoundtripString()
             Assert.Equal("12-34-56-78-9A", BitConverter.ToString(bytes));
             Assert.Equal("56-78-9A", BitConverter.ToString(bytes, 2));
             Assert.Equal("56", BitConverter.ToString(bytes, 2, 1));
-            Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
+
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[0]));
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[3], 1, 0));
```

Comment:
Why is it too strong?  We want our implementations that return empty strings to use the same singleton, and in this case it does.  Doesn't it make sense for the test to flag if that changes in the future?